rotkehlchen/inquirer.py

```diff
@@ -161,14 +164,17 @@ def get_underlying_asset_price(token: EvmToken) -> tuple[Optional[Price], Curren
     """
     price, oracle = None, CurrentPriceOracle.BLOCKCHAIN
     if token.protocol == UNISWAP_PROTOCOL:
-        price = Inquirer().find_uniswap_v2_lp_price(token)
+        price = Inquirer().find_lp_price(token)
```

Comment:
Don't add more elif statmentes. Instead

```python
if token.protocol in (VELODROME_POOL_PROTOCOL, UNISWAP_PROTOCOL):
    price = Inquirer().find_lp_price(token)
    oracle = CurrentPriceOracle.BLOCKCHAIN
```

I believe is fine to go with blockchain here. `UniswapV2` exists because we also have

```
class UniswapV2Oracle(UniswapOracle):
```

that queries prices on chain using the pools but I would standarize this and use blockchain for the location